docs/UserGuide.md

```diff
@@ -110,6 +110,23 @@ Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 
+### **View and edit fields and notes for a person : view**
+
+Edits an existing person's financial information in the address book.
```

Comment:
Consider adding "access client information etc." before edits because now the command is `view` yet the description is only mentions editing.

Reply:
Done!